.github/workflows/ci.yml

```diff
@@ -10,5 +10,11 @@ jobs:
     - uses: actions/checkout@v2
     - name: Build project
       run: swift build
-    - name: Run tests
-      run: swift test
+    - name: Run tests and gather code coverage
+      run: swift test --enable-code-coverage
+    - name: Prepare code coverage
+      run: xcrun llvm-cov export -format="lcov" .build/debug/SwiftInspectorPackageTests.xctest/Contents/MacOS/SwiftInspectorPackageTests -instr-profile .build/debug/codecov/default.profdata > info.lcov
+    - name: Upload code coverage
+      run: bash <(curl https://codecov.io/bash)
+      env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
whaaaa? How does this work?!

Reply:
You configure this from Settings > Secrets 😉